docs/beast_priors.rst

```diff
@@ -32,51 +32,50 @@ The age prior is the star formation rate (SFR) and can be
 
   age_prior_model = {'name': 'flat'}
 
-or
+  or to set the star formation rate in M_sun/year, use
 
-.. code-block:: python
-
-  age_prior_model = {'name': 'flat_linear'}
+  age_prior_model = {'name': 'flat'},
+                     'amp': sfr}
 
 2. Flat in log age
 
 .. code-block:: python
 
   age_prior_model = {'name': 'flat_log'}
 
-3. Set by bins spaced in logage.
+3. Set by bins spaced in logage (log10(years)).
 
 For example, step like priors can be specified by:
 
 .. code-block:: python
 
   age_prior_model = {'name': 'bins_histo',
-                     'logages': [6.0, 7.0, 8.0, 9.0, 10.0],
+                     'x': [6.0, 7.0, 8.0, 9.0, 10.0],
```

Comment:
Again it's "logages", not "x", in the beast_setting file.

Reply:
If we want to change this keyword to "x", the beast_setting file example and  document need to be updated as well.

Reply:
The change to `x` instead of `logage` does not need an update for the beast_settings file example in the `beast-examples` repository as the examples all have `flat` age priors.